document.html

```diff
@@ -738,6 +744,30 @@ <h1 class="large-heading-text margin-1rem">Lists</h1>
 
             </div>
 
+            <hr class="margin-1rem">
+
+            <div class="snackbar-component" id="snackbar">
+                <h1 class="large-heading-text spacing-xs">
+                    Snackbar</h1>
+
+                <p class="small-paragraph-text spacing-s">Snackbars inform users of a process that an app has performed
+                    or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt
+                    the user experience, and they don’t require user input to disappear.</p>
+
+                <p class="small-paragraph-text spacing-s">Use snackbar with property <b>position : fixed</b>
+                </p>
+                <div class="image-box flex-direction-row flex-justify-center border-2px spacing-m">
+
+                    <div class="snackbar">
+                        <h5>Couldn't find the page. Retry in 5 secondss.</h5>
+                        <h5>RETRY</h4>
```

Comment:
Chnage the color of RETRY to slightly light blue. It is hard to see

Reply:
oaky will do